siad/main.go

```diff
@@ -132,6 +132,9 @@ Explorer (e):
 
 // main establishes a set of commands and flags using the cobra package.
 func main() {
+	if build.DEBUG {
+		fmt.Println("Running with debugging enabled")
+	}
```

Comment:
I'm nervous what that would do for compatibility, do not want to change the API. And, just changing the output of `siad version` is not preferred b/c sometimes you aren't sure if a long-running siad instance is debug or not.